packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -47,6 +47,8 @@ class Viewport implements IViewport {
   protected flipHorizontal = false;
   protected flipVertical = false;
   protected rotation = 0;
+  static readonly INITIAL_RANGE = [1.1, 1.1];
+  static readonly CENTER = [0.5, 0.5];
```

Comment:
I don't understand why these are static properties? Do we ever need them without viewport instantiation?

Reply:
They are just the defaults for the values.  Will delete them.